packages/wdio-utils/src/envDetector.js

```diff
@@ -174,14 +172,14 @@ export function sessionEnvironmentDetector ({ capabilities, requestedCapabilitie
  * @param  {Object}  capabilities           caps of session response
  * @return {Object}                         object with environment flags
  */
-export function devtoolsEnvironmentDetector ({ browserName }) {
+export function devtoolsEnvironmentDetector (capabilities) {
     return {
         isDevTools: true,
         isW3C: true,
         isMobile: false,
         isIOS: false,
         isAndroid: false,
-        isChrome: browserName === 'chrome',
+        isChromium: isChromium(capabilities),
```

Comment:
Can we keep `isChrome` to ensure backwards compatibility?

Reply:
if we were to have both would the environment detector functions return isChrome and isChromium.  
It might be best for me to just change the name back to isChrome and avoid all the confusion